src/optimum/rbln/transformers/models/decoderonly/modeling_decoderonly.py

```diff
@@ -22,6 +22,7 @@
 # from Rebellions Inc.
 
 import inspect
+import warnings
```

Comment:
warning 등의 logger가 필요한 경우, built-in warning 대신 utils.logging.py 를 사용해야 합니다. 

```suggestion
import warnings
```